### PR TITLE
cleanup some warnings in the demos

### PR DESCRIPTION
**Pull request by MaximilienNaveau**
Cleaned some warning in the code

### CODE REVIEW

File: demos/const_torque_control.cpp [hunk 1]
```diff
@@ -36,10 +35,9 @@ void ConstTorqueControl::loop()
   while(!stop_loop_)
   {
     time_logger.tic();
-    local_time = count * 0.001;
 
     // compute the control
-    for(int i=0; i<motor_list_.size() ; ++i)
+    for(unsigned int i=0; i<motor_list_.size() ; ++i)
```
Comment (mwuethri) [inline demos/const_torque_control.cpp]: generally, shouldn't we use size_t for that (instead of unsigned?)

Comment (MaximilienNaveau) [reply demos/const_torque_control.cpp]: That's actually a good question... I am not sure.

Comment (mwuethri) [reply demos/const_torque_control.cpp]: i guess it depends actually what we do inside the for loop: https://stackoverflow.com/questions/1951519/should-i-use-stdsize-t-or-int-in-my-for-loops

Comment (MaximilienNaveau) [reply demos/const_torque_control.cpp]: I like the answer, so basically get the type of the comparison constant. size_t for any vector style

**Pull request merged**